.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: SDK Rust CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - '*'
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  test:
+    name: cargo test
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
```

Comment:
![Screenshot 2023-12-01 at 4 52 57 PM](https://github.com/TBD54566975/web5-rs/assets/88001738/0b2584dc-0acb-4fc8-8bf5-6a780a8c7e87)

Currently only take a few seconds longer than macos, so figured it would be fine to have. I'm fine removing though if people don't think it's worth it.